HelpSource/Classes/Buffer.schelp

```diff
@@ -358,7 +358,32 @@ b = Buffer.loadDialog(s, action: { arg buffer;
 )
 x.free; b.free;
 ::
-
+		
+method:: loadChannelDialog
+As link::#*readChannel:: above, but gives you a load dialog window to browse for a file. Cocoa compatible.
```

Comment:
you could drop "Cocoa compatible."? – it should be compatible to all platforms that sclang supports.

Reply:
done! Also removed the corresponding part in `.loadDialog`